style.css

```diff
@@ -0,0 +1,91 @@
+/* Elements Selectors */
+body {
+  background-color: #191765;
+}
```

Comment:
- [ ] [_OPTIONAL_] By default pages have margin and padding, this sometimes might mess up the page. So it's better to remove them and set your own margin and padding. Check this [link ](https://stackoverflow.com/questions/30208335/removing-body-margin-in-css#:~:text=The%20right%20answer%20for%20this%20question%20is%20%22css%20reset%22.&text=It%20removes%20all%20default%20margin,holds%20barred%2C%20regardless%20of%20browser.)for more info